src/components/Firebase/firebase.js

```diff
@@ -65,6 +65,14 @@ class Firebase {
       .add(tournament);
     return result;
   }
+
+  async fetchTournamentById(idTournament) {
+    const result = await this.db.collection(TOURNAMENT_COLLECTION)
+    .where('id', '==', idTournament)
```

Comment:
Is it possible to use `.doc(idTournament)` here instead of a query?

Reply:
We were using an internal id instead of the one provided by firebase, I think we should start using the firebase id and fetch like you say.
There's an issue for that. https://github.com/leusgalvan/fixture/issues/22#issue-479418475 👀